TileFormats/Instanced3DModel/README.md

```diff
@@ -22,61 +22,131 @@ A tile is composed of a header section immediately followed by a body section.
 
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
+
+![body layout](figures/body-layout.png)
 
 ## Header
 
-The 28-byte header contains the following fields:
+The 36-byte header contains the following fields:
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
+| Field name | Data type | Description |
+| --- | --- | --- |
 | `magic` | 4-byte ANSI string | `"i3dm"`.  This can be used to identify the arraybuffer as an Instanced 3D Model tile. |
 | `version` | `uint32` | The version of the Instanced 3D Model format. It is currently `1`. |
 | `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
-| `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates there is not a batch table. |
+| `featureTableJSONByteLength` | `uint32` | The length of the feature table JSON section in bytes. |
+| `featureTableBinaryByteLength` | `uint32` | The length of the feature table binary section in bytes. |
+| `batchTableJSONByteLength` | `uint32` | The length of the batch table JSON section in bytes. Zero indicates that there is no batch table. |
+| `batchTableBinaryByteLength` | `uint32` | The length of the batch table binary section in bytes.
```

Comment:
"The length of the batch table binary section in bytes." -> "The length of the batch table binary section in bytes.  If `batchTableJSONByteLength` is zero; this will also be zero."